Protocol.md

```diff
@@ -166,6 +172,9 @@ The receiving node MUST reject the contract if:
   - The `fund_output_serial_id` and `change_serial_id` are not set to different value
   - Any input in `funding_inputs` is not a BIP141 (Segregated Witness) input.
   - invalid [contract descriptor(s)](./Messaging.md#The-contract_descriptor-Type) are used within `contract_info`.
+  - `cet_locktime` is not set to the earliest maturity time of all included oracle events.
```

Comment:
as I said above, I feel like `cet_locktime` doesn't need to be so strict (and refund locktime should be far enough away from the latest possible execution)